ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/bucketising/GeospatialBucketiser.java

```diff
@@ -32,6 +35,8 @@ public Set<String> bucketise(Member member) {
 					try {
 						return geometryWrapper.convertSRS(SRS_URI.WGS84_CRS);
 					} catch (FactoryException | TransformException e) {
+						LOGGER.error(
+								e.getClass().getName() + " transformed into RdfGeometryException: " + e.getMessage());
```

Comment:
as mentioned in your description.. I don't think we need to log this if we're throwing a new exception for it